README.md

```diff
@@ -49,44 +31,18 @@ Nomi-CEu comes with addon scripts for the following mods. You can drop them into
-
-### Installation Instructions:
-
-#### Script Installation Instructions:
-
-<hr>
-
-##### Windows:
- 1. **(CurseForge Downloads Only)** Download the script [here](https://raw.githubusercontent.com/Nomi-CEu/Nomi-CEu/main/pack-mode-switcher.bat), and save it to the root directory of the pack (the same level as `\config`).
- 2. Run the pack mode switcher in a terminal with `./pack-mode-switcher.bat`
-
-##### Mac/GNU/Linux:
- 1. **(CurseForge Downloads Only)** Run `curl -O https://raw.githubusercontent.com/Nomi-CEu/Nomi-CEu/main/pack-mode-switcher.sh` to download the script, or download it [here](https://raw.githubusercontent.com/Nomi-CEu/Nomi-CEu/main/pack-mode-switcher.sh).
- 2. Run `chmod +x pack-mode-switcher.sh; sh pack-mode-switcher.sh` in the pack root directory (the one containing `/config`).
-
-<hr>
-
-#### Manual installation instructions are available [here](https://github.com/Nomi-CEu/Nomi-CEu/blob/main/overrides/README.md).
-
 ## Credits
-Original pack by [Exaxxion](https://github.com/Exaxxion).  
-Stabilized miner textures from [Self-Torture Edition](https://github.com/NotMyWing/Omnifactory-Self-Torture-Edition).  
-README.md base from [Gregtech CEu](https://github.com/GregTechCEu/GregTech).  
-Certain new quests from [GregTech Community Pack](https://github.com/GregTechCEu/GregTech-Community-Pack).  
-GTCE item-ID to GTCEu item-ID conversion scripts by [brachy84](https://github.com/brachy84).  
-Perfect Gem textures from [FurfSky Reborn](http://furfsky.net/). ([Permission](https://ibb.co/bBpksq0) at [this message](https://discord.com/channels/771187253937438762/774353150278369351/938438074503942184) in [FurfSky Reborn server](https://discord.gg/fsr)).  
-Certain hard mode recipe chains and Magnetron texture from [GCY Science](https://github.com/GregTechCEu/gregicality-science).  
-Flux Networks compatibility recipes by [smudgerox](https://github.com/smudgerox).  
+### Credits from base Nomi-CEu
+- Original pack by [Exaxxion](https://github.com/Exaxxion).  
+- Stabilized miner textures from [Self-Torture Edition](https://github.com/NotMyWing/Omnifactory-Self-Torture-Edition).  
+- README.md base from [Gregtech CEu](https://github.com/GregTechCEu/GregTech).  
+- Certain new quests from [GregTech Community Pack](https://github.com/GregTechCEu/GregTech-Community-Pack).  
+- GTCE item-ID to GTCEu item-ID conversion scripts by [brachy84](https://github.com/brachy84).  
+- Perfect Gem textures from [FurfSky Reborn](http://furfsky.net/). ([Permission](https://ibb.co/bBpksq0) at [this message](https://discord.com/channels/771187253937438762/774353150278369351/938438074503942184) in [FurfSky Reborn server](https://discord.gg/fsr)).  
+- Certain hard mode recipe chains and Magnetron texture from [GCY Science](https://github.com/GregTechCEu/gregicality-science).  
+- Flux Networks compatibility recipes by [smudgerox](https://github.com/smudgerox).  
+
+### Our credits
+- Forked from Nomi-CEu by Integer Limit
```

Comment:
This makes it sound like Integer Limit forked Nomi-CEu to create HOGifactory
Also iirc Integer is not the original creator of nomi ceu, tracer is
Maybe change it something like "Based on [Nomi-CEu](https://github.com/Nomi-CEu/Nomi-CEu/) by the Nomi CEu team"
This could also be something we bring up when talking with them, asking them how they want to be credited

Reply:
yeah you're totally right